datahub-web-react/src/app/entity/user/User.tsx

```diff
@@ -77,6 +77,6 @@ export class UserEntity implements Entity<CorpUser> {
     };
 
     supportedCapabilities = () => {
-        return new Set([]);
+        return new Set([EntityCapabilityType.ROLES]);
```

Comment:
Really like to see us using this EntityCapabilityType infra!

Reply:
Me too, made things easy :)